config/ghostty/config.common

```diff
@@ -0,0 +1,6 @@
+# See also 899be8e51cb288fe0ae91267f495ade7bfb80bc9
+cursor-style-blink = false
+shell-integration-features = no-cursor
```

Comment:
Isn't working to be disabled caused by https://github.com/ghostty-org/ghostty/discussions/2812?